daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -299,6 +299,40 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Consuming from multiple topics
+
+When consuming from multiple topics using a single pub/sub component, there is no guarantee about how the consumers in your consumer group are balanced across the topic partitions. For instance, if you are subscribing to two topics with 10 partitions per topic and you have 20 replicas of your service consuming from the two topics, then there is no guarantee that 10 will be assigned to the first topic and 10 to the second topic. It could be possible that more than 10 are assigned to the first topic and subsequently less than 10 assigned to the second topic. This can result in idle consumers listening to the first topic and over-extended consumers on the second topic, or vice versa. This same beheaviour can be observed when using auto-scalers such as HPA or KEDA.
```

Comment:
```suggestion
When consuming from multiple topics using a single pub/sub component, there is no guarantee about how the consumers in your consumer group are balanced across the topic partitions. 

For instance, let's say you are subscribing to two topics with 10 partitions per topic and you have 20 replicas of your service consuming from the two topics. There is no guarantee that 10 will be assigned to the first topic and 10 to the second topic. Instead, the partitions could be divided unequally, with more than 10 assigned to the first topic and the rest assigned to the second topic. 

This can result in idle consumers listening to the first topic and over-extended consumers on the second topic, or vice versa. This same behavior can be observed when using auto-scalers such as HPA or KEDA.
```

broke up the paragraph a bit, just because it makes it easier for people to read/skim